src/tpm2_wrap.c

```diff
@@ -1063,6 +1068,9 @@ static int wolfTPM2_EncryptSecret_ECC(WOLFTPM2_DEV* dev, const WOLFTPM2_KEY* tpm
     XMEMSET(&eccKeyPriv, 0, sizeof(eccKeyPriv));
     XMEMSET(&pubPoint, 0, sizeof(pubPoint));
     XMEMSET(&secretPoint, 0, sizeof(secretPoint));
+    XMEMSET(&r, 0, sizeof(r));
```

Comment:
`r` is already pointer type, does de-referencing here point XMEMSET to the buffer or a pointer to the buffer?

Reply:
Good catch (this was r directly, then I changed to avoid altering code from r. to r->. I think the behavior is the same. Point to start of array vs pointer to first element in array (same). Pushed fix.